__tests__/app.test.js

```diff
@@ -38,3 +38,57 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an object of the article with the correct article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(typeof article).toBe("object");
+        expect(typeof article.article_id).toBe("number");
+        expect(typeof article.title).toBe("string");
+        expect(typeof article.topic).toBe("string");
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.body).toBe("string");
+        expect(typeof article.created_at).toBe("string");
+        expect(typeof article.votes).toBe("number");
+        expect(typeof article.article_img_url).toBe("string");
+      });
+  });
+  test("200: Responds with an object of the article with the 4th article id", () => {
```

Comment:
as these tests are essentially the same as the first one, they aren't really adding to your testing! So you don't need them 
It would be good to add test to now check what happens if someone tries to use an invalid article id like banana or one that doesnt exist like 1000!